wasm/notebook/src/sample.txt

```diff
@@ -0,0 +1,68 @@
+%%py 
+
+print("Javascript called. It wants its DOM elements back.")
+
+%%md 
+
+## RustPython Notebook with Markdown and Math
```

Comment:
Do you want to have this as the default code? You could do something similar to the demo with the `defaultSnippet` in the html config

Reply:
I will do a different example with default things. I want to add Javascript and css support first. So someone can write js to create like a mini UI. Whatever you select in that UI can then be passed to the python code to do the calculations or modeling. 
I know it is possible because you mentioned vars and js_vars in the previous demo. So, I kinda know how to do that. 
I will wait till I am done with that and then do the sample, I will probably have it be less silly and more boring with the demo all types of mark down elements.